deploy/kubernetes/storage-capacity.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       serviceAccount: csi-provisioner
       containers:
         - name: csi-provisioner
-          image: registry.k8s.io/sig-storage/csi-provisioner:v3.1.0
+          image: registry.k8s.io/sig-storage/csi-provisioner:v3.3.0
```

Comment:
Can we make these templates so we don't need to remember to update the images every release?

Reply:
will open an issue for the improvement/automation @msau42 .

Reply:
@msau42 can we get this in regardless?